docker/Dockerfile

```
@@ -1,72 +1,43 @@
-# use debian as base image
-FROM debian:stretch
+# Use mambaforge as the base image
+# Builds in ~ 4.25 min and is ~ 2.6 GB on a windows laptop
+FROM mambaorg/micromamba:0.15.3
 
 # Label image
 LABEL \
     "Description"="Container for open source time series InSAR processing with Mintpy" \
     "Github Source"="https://github.com/insarlab/MintPy/" \
     "Installation"="https://github.com/insarlab/MintPy/blob/main/docs/installation.md" \
-    "Dockerfile Author"="Andre Theron" \
-    "Email"="andretheronsa@gmail.com"
+    "Dockerfile Author"="Forrest Williams" \
+    "Email"="forrestfwilliams@icloud.com"
 
-# Install useful programs
+# Install command line tools: git, vim and wget
+USER root
 RUN apt-get update && \
-    apt-get install -y \
-    git \
-    python3 \
-    python3-pip \
-    wget \ 
-    vim \
-    zip \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    apt-get install -y git vim wget && \
+    rm -rf /var/lib/apt/lists/*
```

```suggestion
    apt-get install -y --no-install-recommends git vim wget && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

Not necessary, but would help reduce the size of the image and speed up builds.